tests/core/test_processor.py

```diff
@@ -1928,3 +1933,62 @@ async def test_run_anonymization_pipeline_mocked_pipeline(
     await processor.run_anonymization_pipeline(tracker)
 
     event_diff.assert_called_once()
+
+
+async def test_update_full_retrieval_intent(
+    default_processor: MessageProcessor,
+) -> None:
+    parse_data = {
+        "text": "I like sunny days in berlin",
+        "intent": {"name": "chitchat", "confidence": 0.9},
+        "entities": [],
+        "response_selector": {
```

Comment:
Not sure if this is a duplicate curly brace or meant to be a list of dicts instead?

Reply:
It was indeed a duplicate curly brace, I missed it when I copied the response_selector dictionary from an example bot